libs/frontend/feature-build/src/components/toolbar/ToolbarText.vue

```diff
@@ -251,16 +230,7 @@ const iconColor = computed(() => {
   return gradientColors[0]?.rgba ?? color
 })
 
-watch(
-  () => [editor?.value?.selectedComponent, editor.value?.cssPseudoClass],
-  () => {
-    const size = getStyleValue(Css.FontSize)
-    fontSize.value = size?.replace('px', '') ?? ''
-  },
-  {
-    immediate: true,
-  },
-)
+const { fontSizePsInputRef, fontSize, fontSizeSuffix, setFontSize } = useToolbarFontSize()
```

Comment:
This change is intended to solve a warning that occasionally shows up (when a component with non-px font-size is selected).

![Screenshot 2023-11-10 at 10 54 36 AM](https://github.com/samatechtw/pubstudio-builder/assets/59676941/ff36a620-b2ef-48dd-9c25-7951f59116c8)

Turns out it was because we've only handle px in the font-size input in the toolbar, but the font-size of a component can actually be %, em, rem, etc. I create a new composable `useToolbarFontSize` for this. and move all font-size related code from `ToolbarText.vue` to the new composable.

Reply:
Got it!